webapp/src/ts/modules/contacts/contacts-content.component.ts

```diff
@@ -54,7 +54,8 @@ export class ContactsContentComponent implements OnInit, OnDestroy {
   relevantReportForms;
   childContactTypes;
   filteredTasks = [];
-  filteredReports = [];
+  filteredReports: any[] = [];
+  DISPLAY_LIMIT = 50;
```

Comment:
I'm unsure about this. Does this mean that we never display more than 50 reports / tasks in the contact detail page and there is no way that this limit is exceeded?

Reply:
Yes, that is correct. 
- The scope of this issue was to use all reports to calculate the contact-summary. 
- By increasing the limit of reports fetched to 500, we now use "All 500 Reports" to calculate the contact summary 
- A sideeffect of fetching all 500 reports is when a user clicks on View All in the contact detail page, we do not have pagination or lazy loading, the app might freeze as it fetches all 500. 
- Therefore, we limited the number of reports to 50 whether you are in the 3 month, 6 month or View all tab.  If a user wants to View All reports they have to visit the Reports tab. 
- We could create another issue to add pagination for the View All tab to efficiently load All reports there.

Reply:
Cool, thanks for the explanation. I agree that we need a new issue to add pagination to tasks / reports on contact detail pages.